common/stash/pom.xml

```diff
@@ -72,16 +72,4 @@
 
     </dependencies>
 
-    <build>
-        <plugins>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-surefire-plugin</artifactId>
-                <configuration>
-                    <!-- Need to add system property to force default character encoding to be US-ASCII -->
-                    <argLine>@{argLine} -Dfile.encoding=US-ASCII</argLine> <!-- activate the JaCoCo javaagent -->
```

Comment:
we might need to leave the plug-in itself, and remove just `@{argLine}`

Reply:
this plugin was restored